src/bp/core/modules/resources-loader.ts

```diff
@@ -1,13 +1,12 @@
 import { Logger } from 'botpress/sdk'
```

Comment:
What do you think of `resources-loader.ts` -> `module-resource-loader.ts` to fit with the name of the class?